examples_source/1D_simulation(macro_amorphous)/plot_0_protein_GB1.py

```diff
@@ -26,8 +26,9 @@
 # Create the Simulator object and load the spin systems from an external file.
 sim = Simulator()
 
-file_ = "https://sandbox.zenodo.org/record/835664/files/protein_GB1_15N_13CA_13CO.mrsys"
-sim.load_spin_systems(file_)  # load the spin systems.
+host = "https://ssnmr.org/sites/default/files/mrsimulator/"
```

Comment:
why do you have the host and filename separate in some changes and all in one line in others?

Reply:
@mccarthy677 Flake8, the Python code linter, has an 88 character per line limit and some of the urls on their own surpass 88 characters. When you run `pre-commit run` the line lengths are checked and Flake8 will get be unhappy if any are over. The splits are simply to adhere to standard Python convention